sigs.yaml

```diff
@@ -2930,16 +2930,22 @@ workinggroups:
       company: Intel
   meetings:
   - description: Regular Meeting
```

Comment:
nit: put the calendar in both descriptions

Reply:
yup, fixed.